website/docs/zh/guide/basic/cli.mdx

```diff
@@ -1 +1,232 @@
 # 命令行工具
+
+Rslib 提供了一个轻量级的命令行工具，包含 [rslib build](#rslib-build) 和 [rslib inspect](#rslib-inspect) 等命令。
+
+## rslib -h
+
+如果你需要查看所有可用的 CLI 命令，请在项目目录中运行以下命令：
+
+```bash
+npx rslib -h
+```
+
+输出如下：
+
+```text
```

Comment:
I see that this section is also not translated to chinese in Rsbuild doc, is that meet our expect?

Reply:
I will translate it

Reply:
done